client/src/app/site/pages/meetings/pages/motions/components/motion-forward-dialog/components/motion-forward-dialog/motion-forward-dialog.component.html

```diff
@@ -1,55 +1,134 @@
-                                [(ngModel)]="checkboxStateMap[meeting.id]"
-                                (change)="onChangeCheckbox($event)"
-                            >
-                                {{ meeting.name }}
-                                @if (meeting.start_time || meeting.end_time) {
+    <table mat-table [dataSource]="['motion_version', 'submitter', 'identifier', 'meeting']">
+        <!-- Label in front-->
+        <ng-container matColumnDef="label">
+            <td *matCellDef="let rowName" mat-cell>
+                @if (rowName === 'motion_version') {
+                    <span>{{ 'Motion version' | translate }}</span>
+                }
+                @if (rowName === 'submitter') {
+                    <span>{{ 'Submitter' | translate }}</span>
+                }
+                @if (rowName === 'identifier') {
+                    <span>{{ 'Identifier' | translate }}</span>
+                }
+                @if (rowName === 'meeting') {
+                    <span>{{ 'Meetings' | translate }}</span>
+                }
+            </td>
+        </ng-container>
+
+        <!-- radio checks -->
+        <ng-container matColumnDef="radio">
+            <td *matCellDef="let rowName" mat-cell>
+                @if (rowName === 'motion_version') {
+                    <span>
+                        <mat-radio-group class="motion-forward-radio-group" [(ngModel)]="useOriginalVersion">
+                            <mat-radio-button value="true">
+                                {{ 'Original motion text with amendments/change recommendations' | translate }}
+                            </mat-radio-button>
+                            <mat-radio-button value="false">
+                                {{ 'Final version' | translate }}
+                            </mat-radio-button>
+                        </mat-radio-group>
+                    </span>
+                }
+                @if (rowName === 'submitter') {
+                    <span>
+                        <mat-radio-group class="motion-forward-radio-group" [(ngModel)]="useOriginalSubmitter">
+                            <mat-radio-button value="true">
+                                {{ 'original submitter name' | translate }}
+                                @if (data.motion.length === 1 && data.motion[0].submitterNames.length > 0) {
+                                    <span>
+                                        <os-comma-separated-listing [list]="data.motion[0].submitterNames">
+                                            <ng-template let-submitter>
+                                                <b>{{ submitter }}</b>
+                                            </ng-template>
+                                        </os-comma-separated-listing>
+                                    </span>
+                                }
+                                @if (data.motion.length === 1 && data.motion[0].submitterNames.length === 0) {
```

Comment:
Maybe use `@else if (...)`

Reply:
This might be applicable in other places too.

Reply:
done in the dialog.